processing/src/main/java/io/druid/segment/virtual/SingleLongInputCachingExpressionColumnValueSelector.java

```diff
@@ -150,6 +150,6 @@ private ExprEval eval(final long value)
   @Override
   public boolean isNull()
   {
-    return getObject().isNull();
+    return getObject().isNumericNull();
```

Comment:
Why this?

Reply:
It is possible for an expression to have a non-null String value but it can return null when parsed as a primitive long/float/double.
ExprEval.isNumericNull checks whether the parsed primitive value is null or not.
Added comment to code.